fiftyone/utils/ultralytics.py

```
@@ -1055,3 +1187,30 @@ def __call__(
+    boxes[:, 1] *= img_height
+    boxes[:, 3] *= img_height
+
+    labels = [d.label for d in dets]
+    classes = list(dict.fromkeys(labels))
+    label_to_idx = {c: i for i, c in enumerate(classes)}
+    cls_indices = [label_to_idx[label] for label in labels]
+
+    return boxes.tolist(), cls_indices, classes
+
+
+def _get_yoloe_vp_predictor():
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_get_yoloe_vp_predictor`

(ANN202)